### PR TITLE
Add a possibility to create a view backend before creating a View

### PR DESCRIPTION
**Pull request by Gavriliuk**
Additionally, small bugs fixed (avoid of using uninitialized nonzero pointers)

### CODE REVIEW

File: Source/WebKit2/UIProcess/API/C/wpe/WKView.h [hunk 1]
```diff
@@ -36,6 +36,7 @@ extern "C" {
 struct wpe_view_backend;
 
 WK_EXPORT WKViewRef WKViewCreate(WKPageConfigurationRef);
+WK_EXPORT WKViewRef WKViewCreateWithDefaultViewBackend(WKPageConfigurationRef);
```
Comment (vertexodessa) [inline Source/WebKit2/UIProcess/API/C/wpe/WKView.h]: maybe WKViewCreateCreatingBackendFirst would be more descriptive

Comment (Gavriliuk) [inline Source/WebKit2/UIProcess/API/C/wpe/WKView.h]: I agree
Maybe even WKViewCreateCreatingViewBackendFirst ?

**Comment by zdobersek** [comment]
There's no need for this API. There's already WKViewCreateWithViewBackend(). It's trivial for the user to create a view backend manually and pass it to that.

**Comment by Gavriliuk** [comment]
If there were no need to this API we wouldn't have implemented it.

Creating of a ViewBackend has been implemented in libWPE.
The user is not required to know how a backend is being created.
He only wants to use a special creation order to setup env before starting WebProcess.

To create a backend manually user has to link his app with libWPE and its include files.
That will create extra dependencies.
And when the creation method will be changed in future, user must to modify his app too.
That will create extra dependencies too.

We tried this way first but it looked as a very heavy solution.
It will be much more useful to add such a new function in WebKit and not to create extra dependencies.
We prefer to minimize changes in sources and in their structure.
Such a new function won't bother anyone but can really simplify our work.

**Comment by zdobersek** [comment]
Setting up the environment early enough in the UIProcess is simple to achieve by using the wpe_renderer_host interface, which is initialized before any WebProcess is created. There's no need for a 'special creation order'.

Simply put, view backend is no place to set up process-wide state, so I would recommend against it. But since this isn't a pull request against the master branch, where my priorities lie, we can merge this.

**Pull request merged**